py/modsys.c

```diff
@@ -180,6 +209,14 @@ STATIC const mp_rom_map_elem_t mp_module_sys_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_exit), MP_ROM_PTR(&mp_sys_exit_obj) },
     #endif
 
+    #if MICROPY_PY_SYS_UATEXIT
+    { MP_ROM_QSTR(MP_QSTR_uatexit), MP_ROM_PTR(&mp_sys_uatexit_obj) },
+    #endif
+
+    #if MICROPY_PY_SYS_TRACE
```

Comment:
Since the function is `settrace` the config option should really be `MICROPY_PY_SYS_SETTRACE` (to be consistent with the other options).